package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.2",
   "license": "MIT",
   "description": "Autoreply for Messenger",
-  "author": "Emma Arfelt",
+  "author": "Emma Arfelt <trordu@selv.dk>",
```

Comment:
An email is needed in order to the debian package system to accept the package (you need to know the maintainer)
https://www.debian.org/doc/manuals/debian-faq/ch-pkg_basics.en.html#s-deb-format